features/step_definitions/administration.rb

```diff
@@ -1,99 +1,110 @@
 Given(/^I have an admin account$/) do
-  pending # express the regexp above with the code you wish you had
+ @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
 end
 
 When(/^I visit the admin panel url$/) do
-  pending # express the regexp above with the code you wish you had
+  visit("/admin") 
 end
 
 When(/^I enter my admin email$/) do
-  pending # express the regexp above with the code you wish you had
+  fill_in :admin_user_email, with: "admin@example.com" 
 end
 
 When(/^I enter my admin password$/) do
-  pending # express the regexp above with the code you wish you had
+  fill_in :admin_user_password, with: "password" 
 end
 
 Then(/^I see the admin panel$/) do
-  pending # express the regexp above with the code you wish you had
+  expect(page).to have_content "Signed in successfully" 
 end
 
 Given(/^I do not have an admin account$/) do
-  pending # express the regexp above with the code you wish you had
+  AdminUser.delete_all
 end
 
 Then(/^I see a flash notification that tell me that my email does not exist in the system$/) do
-  pending # express the regexp above with the code you wish you had
+  expect(page).to have_content "Invalid email or password" 
 end
 
 Given(/^I am logged into the admin panel$/) do
-  pending # express the regexp above with the code you wish you had
-end
-
-Given(/^I am logged into the site$/) do
-  pending # express the regexp above with the code you wish you had
+  @admin = AdminUser.create(email: "admin@example.com", password: "password", password_confirmation: "password")
+  visit("/admin")
+  fill_in :admin_user_email, with: "admin@example.com"
+  fill_in :admin_user_password, with: "password"
+  click_button("Login") 
 end
 
 When(/^I visit the admin books url$/) do
-  pending # express the regexp above with the code you wish you had
+  visit("/admin/books") 
 end
 
 When(/^I enter the title "(.*?)"$/) do |arg1|
-  pending # express the regexp above with the code you wish you had
+  fill_in :book_title, with: arg1 
```

Comment:
don't use the default `arg1` param name cucumber gives you. make it meaningful. In this case, `title` is more appropriate.